rules/windows/discovery_file_dir_discovery.toml

```diff
@@ -21,6 +21,37 @@ index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "File and Directory Discovery"
+note = """## Triage and analysis
+
+### Investigating File and Directory Discovery
+
+After a successful initial compromise, attackers may try to gain situational awareness of the environment to plan their
+next steps. This can happen by running commands to enumerate network resources, users, connections, files, and security
+software installed.
```

Comment:
```suggestion
next steps. This can happen by running commands to enumerate network resources, users, connections, files, and installed security
software.
```